### ISSUE TITLE
About the mIoU metric

### ISSUE THREAD

**Issue by JunXieFront** [microsoft/ProDA#45]
The warmup pretrained model was supposed to be 43.3 mIoU, but when I test on cityscapes valid dataset, I've got a really lower one.I'm using the runningSocre class in your Repo and my metric function with the same result.
![image](https://user-images.githubusercontent.com/49926773/162553110-d70d1c2e-dc81-49dd-9adf-9ecd72de9acc.png)
I'm really confused with the result, could you tell me how to get the correct mIoU about the pretrained model?

**Comment by JunXieFront** [comment]
Maybe the version of PyTorch caused the unexpected result
